src/js/types/generic.type.ts

```diff
@@ -17,3 +17,5 @@ export type PlayerAPI = any;
 export type PlaylistItemSource = any;
 
 export type DefaultProvider = any;
+
+export type TextTrackLike = any;
```

Comment:
i assume this is different than `item/track.ts`?

Reply:
It comes primarily from https://github.com/jwplayer/jwplayer/blob/master/src/js/providers/tracks-mixin.js#L232 where most of the time the provider is using native textTracks. It's not from the playlist item. It should be typed once that mixin is typed.

Reply:
can we add this to maria's ts doc where we're tracking this stuff? https://jwplayer.atlassian.net/wiki/spaces/JW8/pages/607453187/Typescript+Standards

thanks!

Reply:
I don't see any of these in that doc, and they are all tech-debt. Why don't we just comment above each of these type definitions rather than keeping another document?

```
export type CoreModel = any;

export type PlaylistItemType = any;

export type HTMLTemplateString = string;

export type PlayerAPI = any;

export type PlaylistItemSource = any;

export type DefaultProvider = any;

export type Menu = GenericObject;

export type Tooltip = GenericObject;

export type Localization = GenericObject;

export type TextTrackLike = TextTrack & {
    _id: string;
    data: Array<any>;
    name: string;
};
```

Reply:
For things in one file I think it's easier but my concern with doing comments outside this case is they'd be impossible to find after. Maybe we do `// TS:` so it's easily searchable and scrap the doc?

Reply:
Isn't this that one file?

Reply:
It _should_ be, the doc was meant to catch quirks or problems that might surface beyond having shared untyped generics here but maybe that's overengineering